packages/core/src/App/Components/Elements/Modals/deriv-real-account-required-modal.jsx

```diff
@@ -47,13 +50,16 @@ DerivRealAccountRequiredModal.propTypes = {
     enableApp: PropTypes.func,
     openRealAccountSignup: PropTypes.func,
     is_eu_user: PropTypes.string,
+    is_traders_hub: PropTypes.bool,
```

Comment:
Please remove this as its not used anywhere

Reply:
Already done 👍🏼